spec/requests/posts_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts', type: :request do
+  describe 'GET /posts' do
+    before(:example) do
+      get '/users/1/posts'
```

Comment:
This test is for the Post index page, you are missing a required Post show page. That is with the route `/users/:id/posts/:id`. Kindly include a test for this.